actions/banked-months-updater.vbs

```diff
@@ -0,0 +1,482 @@
+	Call write_variable_in_SPEC_MEMO("Unless you or someone in your SNAP unit meet work rules or an exemption, you/they will no longer be eligible for SNAP.")
+	Call write_variable_in_SPEC_MEMO("")
+	Call write_variable_in_SPEC_MEMO("However, due to additional funding we are able to approve SNAP benefits for up to 2 more months.")
+	Call write_variable_in_SPEC_MEMO("")
+	Call write_variable_in_SPEC_MEMO("If you/someone in your SNAP unit is not meeting work requirements/meeting an exemption, you/they will no longer receive SNAP after these 2 months.")
+	Call write_variable_in_SPEC_MEMO("")
+	Call write_variable_in_SPEC_MEMO("Please contact your team if you, or someone in your SNAP unit, start meeting work requirements or think that you meet an exemption.")
+	Call write_variable_in_SPEC_MEMO("")
+	Call write_variable_in_SPEC_MEMO("If you need help meeting these work requirements, please see the SNAP Time-limited work rules website at: https://mn.gov/dhs/snap-e-and-t/time-limited-work-rules/.")
+	Call write_variable_in_SPEC_MEMO("************************************************************")
+	PF4 'save memo
+	stats_counter = STATS_counter + 1
+End if 
+
+'----------------------------------------------------------------------------------------------------CASE/NOTE
+Call start_a_blank_CASE_NOTE
+Call write_variable_in_case_note("--SNAP Banked Months Evaluation for " & initial_month & "/" & initial_year & "--")
+Call write_variable_in_case_note("Case TLR Information by Member")
+Call write_variable_in_case_note("------------------------------")
+For i = 0 to Ubound(banked_months_array, 2)
+    Call write_variable_in_case_note(banked_months_array(member_first_name_const, i) & " (MEMB " & banked_months_array(member_number_const , i) & ")")
+    Call write_variable_in_case_note("* ABAWD Count/Months Used: " & banked_months_array(abawd_mo_count_const, i) & " - " & banked_months_array(abawd_mo_string_const, i))
+    Call write_variable_in_case_note("* ABAWD Months Evaluation: " & banked_months_array(abawd_month_eval_const, i))
+    Call write_variable_in_case_note("* Banked Count/Months Used: " & banked_months_array(banked_mo_count_const, i) & " - " & banked_months_array(banked_mo_string_const, i))
+    Call write_variable_in_case_note("--")
+	stats_counter = STATS_counter + 1
+Next 
+If update_wreg = True then Call write_variable_in_case_note("* The STAT/WREG panel was updated through CM + 1 for members who have banked months available, or who have exhausted them.")
+If spec_memo = True then Call write_variable_in_case_note("* A SPEC/MEMO was sent to the household with banked months and time-limited SNAP information.")
+Call write_variable_in_case_note("---")
+CALL write_variable_in_CASE_NOTE(worker_signature)
+stats_counter = STATS_counter + 1
+
+end_msg = "The main EATS household has been assessed and updated for banked months. Make sure to APP new SNAP results, and use NOTES - ELGIBILTY SUMMARY to case note the eligibility."
```

Comment:
Small typo (ELGIBILTY -> ELIGIBILITY)

Reply:
Do you like how I spelled it wrong and right in the same sentence @MDR22?